api/src/main/java/org/openmrs/module/fhir/api/impl/LocationServiceImpl.java

```diff
@@ -135,5 +135,24 @@ public void updateLocationById(String id, Location location) {
 		}
 		Context.getLocationService().saveLocation(omrsLocation);
 	}
+	
+	/**
+	 * @see org.openmrs.module.fhir.api.LocationService#createLocation(ca.uhn.fhir.model.dstu2.resource
+	 * .Location)
+	 */
+	public Location createLocation(Location location) {
+		org.openmrs.Location omrsLocation=null;
+		List<String> errors = new ArrayList<String>();
+		omrsLocation = FHIRLocationUtil.generateOpenMRSLocation(location, errors);
+		if (!errors.isEmpty()) {
+			StringBuilder errorMessage = new StringBuilder("The request cannot be processed due to the following issues \n");
+			for (int i = 0; i < errors.size(); i++) {
+				errorMessage.append((i + 1) + " : " + errors.get(i) + "\n");
+			}
+			throw new UnprocessableEntityException(errorMessage.toString());
+		}
+		omrsLocation=Context.getLocationService().saveLocation(omrsLocation);
```

Comment:
please make sure you are using proper code formatter. We are always leaves blank space before and after equal sign. Please address this in separate pull request.